tests/gis_tests/geos_tests/test_io.py

```diff
@@ -115,10 +126,11 @@ def test04_wkbwriter(self):
 
         # Ensuring bad output dimensions are not accepted
         for bad_outdim in (-1, 0, 1, 4, 423, "foo", None):
-            with self.assertRaisesMessage(
-                ValueError, "WKB output dimension must be 2 or 3"
-            ):
-                wkb_w.outdim = bad_outdim
+            with self.subTest(bad_outdim=bad_outdim):
+                with self.assertRaisesMessage(
+                    ValueError, "WKB output dimension must be 2 or 3"
+                ):
```

Comment:
:wave: @smithdc1 do you fancy parenthesized context managers as a means to reduce indentation?